pkg/cmd/roachtest/tests/failover.go

```diff
@@ -717,6 +726,52 @@ func (f *crashFailer) Recover(ctx context.Context, nodeID int) {
 	f.c.Start(ctx, f.t.L(), f.startOpts, f.startSettings, f.c.Node(nodeID))
 }
 
+// diskStallFailer stalls the disk indefinitely. This should cause the node to
+// eventually self-terminate, but we'd want leases to move off before then.
+type diskStallFailer struct {
+	t             test.Test
+	c             cluster.Cluster
+	m             cluster.Monitor
+	startOpts     option.StartOpts
+	startSettings install.ClusterSettings
+}
+
+func (f *diskStallFailer) Setup(ctx context.Context) {
```

Comment:
Could you add a comment here?

Does this work on AWS and GCE alike? (No need to check, just eyeball it - for now this runs under GCE anyway)

Reply:
I'll have to check that the AWS images contain `dmsetup`. If it does, it should work, but I'll verify.

Reply:
AWS uses a different device path, but it works once that's fixed.